cluster/gce/configure-vm.sh

```diff
@@ -661,7 +688,17 @@ if [[ -z "${is_push}" ]]; then
   remove-docker-artifacts
   run-salt
   set-good-motd
+
+  curl-metadata k8s-user-startup-script > ${INSTALL_DIR}/k8s-user-script.sh
```

Comment:
Actually, we definitely need to check here, otherwise a non-zero return value will cause bash to exit prematurely.